example-code/snippets/src/pcf/DynamicSchema.ts

```diff
@@ -129,25 +129,19 @@ async function createDynamicSchema(
 
   const createEntityClass = async (schema: MetaSchema) => {
     for (const entity of map.entities) {
-      const entityResult = await editor.entities.createFromProps(schema.schemaKey, entity.props);
-      if (!entityResult.itemKey)
-        throw new Error(`Failed to create EC Entity Class - ${entityResult.errorMessage}`);
+      const entityKey = await editor.entities.createFromProps(schema.schemaKey, entity.props);
 
       if (entity.props.properties) {
         for (const prop of entity.props.properties) {
-          const propResult = await editor.entities.createPrimitiveProperty(entityResult.itemKey, prop.name, prop.type as any);
-          if (!propResult.itemKey)
-            throw new Error(`Failed to create EC Property - ${propResult.errorMessage}`);
+          await editor.entities.createPrimitiveProperty(entityKey, prop.name, prop.type as any);
         }
       }
     }
   };
 
   const createRelClasses = async (schema: MetaSchema) => {
```

Comment:
is this fn even needed anymore?

Reply:
Do you mean the example-code?  If so, not sure.  the createPrimitiveProperty method still exists, but we now throw errors instead of 'result' objects.

Reply:
Sorry mis-read your comment. Yes, the function is still needed.